aws.yaml

```diff
@@ -1,4 +1,101 @@
 - active: false
+  apps:
+    - app: chart-operator
+      version: 0.11.2
+      use-upgrade-force: true
```

Comment:
This is configuration that shouldn't be in the releases repo. We use upgrade force for all apps except coredns and nginx-ingress-controller. It isn't safe for these apps as a forced upgrade could cause downtime.

Reply:
I would delete `use-upgrade-force` and put them as a same way we would put `namespace` information inside `/pkg/`.

Reply:
Yes I think having this as separate metadata is better. The releases YAML should just be about the components and versions.

I think cluster-operator/pkg is a good home but it could also be a apps.yaml file in this repo. As we want to keep the cluster-operator changes to a minimum. WDYT?

Reply:
On 2nd thoughts I don't like the apps.yaml idea. I don't think apps metadata should be in this repo. 

@tomahawk28 How about starting with `cluster-operator/pkg/apps`?

The priority is to make updating the versions easier. The apps metadata changes much less often.